src/consensus/vote.cpp

```diff
@@ -570,8 +570,13 @@ std::string VoteManager::getJsonStr(std::vector<Vote> const& votes) {
   return ptroot.toStyledString();
 }
 
-NextVotesForPreviousRound::NextVotesForPreviousRound(addr_t node_addr, std::shared_ptr<DbStorage> db)
-    : db_(db), enough_votes_for_null_block_hash_(false), voted_value_(NULL_BLOCK_HASH), next_votes_size_(0) {
+NextVotesForPreviousRound::NextVotesForPreviousRound(addr_t node_addr, std::shared_ptr<DbStorage> db,
+                                                     std::shared_ptr<FinalChain> final_chain)
+    : db_(db),
+      final_chain_(final_chain),
```

Comment:
use std::move for both db and final_chain shared pointers

Reply:
Fixed, thanks